.gitignore

```diff
@@ -46,3 +46,6 @@ slither/
 
 # Mac
 .DS_Store
+
+# Misc
+.vscode/
```

Comment:
I believe we adopt a pattern where .vscode is globally ignored in the developer machine, don't we? Still, it doesn't hurt to be here. Should we put the folder .idea (Jetbrains) here too?

Reply:
It was still coming up for me, so it must have gotten overwritten. I don't think it hurts, and can add .idea too.